readthedocs/restapi/views/model_views.py

```diff
@@ -260,6 +260,11 @@ def get_queryset(self):
         org = self.request.query_params.get('org', None)
         if org is not None:
             query = query.filter(organization__pk=org)
+
+        own = self.request.query_params.get('own', None)
+        if own is not None:
+            query = query.filter(organization__isnull=True)
```

Comment:
I think it can be written something like
``query = query.filter(organization=None)``